medusa/tv/series.py

```diff
@@ -2160,11 +2162,20 @@ def want_episode(self, season, episode, quality, forced_search=False, download_c
         should_replace, reason = Quality.should_replace(ep_status, cur_quality, quality, allowed_qualities,
                                                         preferred_qualities, download_current_quality,
                                                         forced_search, manually_searched)
-        if reason == 'leaked':
+        if reason == 'leaked' and not notified_earlier_release:
             notify_message = '{show} {ep} with quality: {quality}'.format(show=self.name,
                                                                           ep=episode_num(season, episode),
                                                                           quality=Quality.qualityStrings[quality])
             notifiers.notify_leaked(notify_message)
+
+            # Update notify flag
+            main_db_con.action(b'UPDATE tv_episodes '
+                               b'SET notified_earlier_release = 1 '
+                               b'WHERE '
+                               b'  showid = ? '
+                               b'  AND season = ? '
+                               b'  AND episode = ?', [self.indexerid, season, episode])
+
```

Comment:
my only concern about this is that if we need to update episode object loaded from DB or not
No idea